File: synthesejava/Case.java
package synthesejava;

import java.awt.Dimension;
import java.awt.Point;

public class Case {
    private Point pos;
    private static Dimension size;

    public Case(Point pos) {
        this.pos = pos;
    }

    public void setPos(Point pos) {
        this.pos = pos;
    }

    public Point getPos() {
        return pos;
    }

    public static void setSize(Dimension size) {
        Case.size = size;
    }

    public static Dimension getSize() {
        return size;
    }
}

File: Case.cpp
#include "Case.h"

QSize Case::size = QSize(5, 5);

Case::Case(QPoint pos){
	this->pos = pos;
}

Case::~Case(){
}

QPoint
Case::getPos(void){
	return this->pos;
}

QRect
Case::getRect(void){
	return QRect(this->pos, this->size);
}

QSize
Case::getSize(void){
	return this->size;
}
